evcouplings/compare/sifts.py

```diff
@@ -137,8 +140,37 @@ def find_homologs_jackhmmer(**kwargs):
     if config["prefix"] is None:
         config["prefix"] = path.join(tempdir(), "compare")
 
-    # run jackhmmer against sequence database
-    ar = jackhmmer_search(**config)
+    check_required(
+        config,
+        [
+            "database", "prefix", "use_bitscores",
+            "domain_threshold", "sequence_threshold",
+            "nobias", "cpu",
+        ]
+    )
+
+    # run hmmsearch (possibly preceded by hmmbuild)
+    if not use_jackhmmer:
+        hmmsearch_result = hmmbuild_and_search(
+            database = config["sequence_database"],
+            prefix = config["prefix"],
+            use_bitscores = config["use_bitscores"],
+            domain_threshold = config["domain_threshold"],
+            seq_threshold = config["sequence_threshold"],
+            nobias = config["nobias"],
+            cpu = config["cpu"],
+            binary = [path.join(config["hmmer"], "hmmbuild"), 
+                        path.join(config["hmmer"], "hmmsearch")],
+        )
+
+        ar = {
```

Comment:
What comes out of hmmbuild_and_search() should be the same data structure that comes out of jackhmmer-search() (jackhmmer is essentially a specialized case of hmmbuild/hmmsearch)